azure-toolkit-libs/azure-toolkit-springcloud-lib/src/main/java/com/microsoft/azure/toolkit/lib/springcloud/AzureSpringCloudConfigUtils.java

```diff
@@ -167,4 +173,19 @@ public static UserSourceInfo getOrCreateSource(
         }
         return source;
     }
+
+    public static boolean isEmpty(ScaleSettings scale) {
+        return Objects.isNull(scale.getCapacity()) && Objects.isNull(scale.getMemoryInGB()) && Objects.isNull(scale.getCpu());
+    }
```

Comment:
what's the meaning of "isEmpty" ? it seems belong to ScaleSettings class. not same with SpringCloud, right?